rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -13,15 +18,19 @@
 
 
 @pytest.mark.vcr()
-@pytest.mark.parametrize('ethereum_accounts', [['0x648aA14e4424e0825A5cE739C8C68610e143FB79']])
+# Analyze EOA and user's DSProxy address
```

Comment:
no this is not the way. 

1. We don't put comments here, but only in docstrings of test
2. We never manually track DSProxies. This is not the solution. The solution is to make it get detected by rotki. Make sure the code that detects the proxies runs.

Reply:
2. That currently doesn't work